### DISCUSSION TITLE
Tagging saved notes like in telegram but simpler

### DISCUSSION THREAD

**Discussion by Assurokilwr** [stoatchat/discussions#707] [Feature Suggestions]
https://telegram.org/blog/new-saved-messages-and-9-more
Just make it like [standard notes](https://play.google.com/store/apps/details?id=com.standardnotes) tags where the messages indexed at the client and sorted by tags one by one
![SmartSelect_20240328_172322_Obsidian](https://github.com/revoltchat/revolt/assets/165287756/362de3e2-c01e-4d05-90b8-a1f7fb3a223a)
So basically the tags would be a fake folders

**Comment by rogue-agent** [comment]
Would one be able to tag any message? And isn't it just bookmarking with folders?
